### PR TITLE
Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lisaychuang**
Updated CONTRIBUTING.md file to include:
* link to installation guide
* grammar / contact me details
* Added a screenshot for Mac gcc installation using Homebrew.

### CODE REVIEW

**Comment by emhoracek** [comment]
Oh my this is looking amazing! I love the check list. It made me realize I forgot two dependencies:

NetPBM: this is how the images are converted from the *.pnm format that `cel2pnm` creates into a PNG image that a browser can display. You can install it on Mac with `brew install netpbm`.

LHA: this is the archive fomat that KiSS dolls use. Unfortunately, I can't remember where I found the program and got it to work -- it's so old!  I'll figure it out and report back.

Sorry this project is such a pain! I hope to simplify it over time! Thanks for your efforts!!

**Comment by lisaychuang** [comment]
yay! Before I update the **General Packages** requirement, can you confirm if both NetPBM & LHA are already installed through stack build / stack set up? Or if I should add them as separate packages to install? Thanks Libby :)

**Comment by emhoracek** [comment]
Unfortunately, both of these are separate packages that need installed separately. NetPBM is usually available via apt-get or other package repositories, so it's pretty easy to install. 

LHA may be harder. I originally downloaded it from here: http://freecode.com/projects/lhaforunix (Right-click save-as the button that say 1.17) Then I had to unzip the file, cd into the directory, and run `make` and `make install`. Just including LHA in the Smooch project would make it easier, but the license may prohibit that. I'll do some more research!

**Comment by lisaychuang** [comment]
Just added **Install Netpbm** section, do you want to merge this PR first? And we can continue the read on Issue #23 to resolve `$PATH` and `LHA` installation sections.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,36 +108,91 @@ Items maked a `*` will need work on the frontend as well.
...
+* Next, run `stack build` which will install project dependencies
+![alt text](https://preview.ibb.co/iC64ow/stack_build.png "stack build")
 
-Once you have `stack` installed, change to the `haskell` directory and run
-`stack setup`. That will install the correct Haskell version (this will take a
-while if you don't already have it).
+* Next, run `stack install rivet-autoimporter`
 
-Next, run `stack build`. That will compile the Smooch app.
+That will compile the Smooch app.
```
Comment (emhoracek) [inline CONTRIBUTING.md]: `stack build` is the part that compiles the Smooch app, `rivet-autoimporter` is just another dependency, a tool for migrating the database. So, I think the `stack installl rivet-autoimporter` step should go above the `stack setup` step so this sentence makes more sense.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -108,36 +108,91 @@ Items maked a `*` will need work on the frontend as well.
...
+## General Packages
+
+Please make sure you have the following general software installed
+
+| Required | Library | Version Range | Notes |
+| ------------- | ------------- | ---| --- |
+| ✔ | [GCC compiler](https://gcc.gnu.org/)  | >= 7.2 | GCC is the compiler for [GNU operating system](http://www.gnu.org/gnu/thegnuproject.html), which includes C, C++, Objective-C, Fortran, Ada, and Go |
+| ✔ | [stack](https://docs.haskellstack.org/en/stable/README/#the-haskell-tool-stack)  | >= 1.5.1 | `stack` is a Haskell dependency management tool |
+| ✔ | [Netpbm](http://brewformulas.org/Netpbm)  | >= 10.73.14 | Netpbm is a toolkit for manipulation of graphic images, including conversion of images between a variety of different formats |
+|  | [Homebrew](https://brew.sh/)  | >= 1.3.4 | Homebrew is the missing package manager for macOS |
```
Comment (emhoracek) [inline CONTRIBUTING.md]: Really like this idea!

**Comment by emhoracek** [comment]
@lisaychuang Good idea! I went through and reviewed -- found one minor thing that it would be great if you could fix before I merge. (I didn't specify order when I asked you to add the autoimporter step, I know, my bad)

**Comment by lisaychuang** [comment]
Done!

**Comment by emhoracek** [comment]
Thanks so much! I really appreciate the work you've done here, and other contributors will appreciate it too! Would you like commit access to the project?

**Comment by lisaychuang** [comment]
Yay! Will be an honor to get commit access, I'll love to see how this project grows :)

**Pull request merged**